src/views/Timesheets/index.js

```diff
@@ -582,209 +599,233 @@ const Timesheets = (props) => {
       </Button>
     );
 
-    return networkStatus === 'online' ? (
-      <>
-        <Grid container spacing={2}>
-          <Grid item xs={12}>
-            <MuiPickersUtilsProvider utils={DateFnsUtils}>
-              <Card>
-                <CardContent
-                  style={{
-                    marginLeft: 8,
-                    marginTop: 8,
-                  }}
-                >
-                  <Grid container spacing={2} alignItems="center" alignContent="center">
-                    <Grid item md={2}>
-                      <Button onClick={changeState}> {clockInOut}</Button>
-                    </Grid>
-                    <Grid item md={8}>
-                      <div className="header-tooltip-container">
-                        <CustomTooltip
-                          classes={{ tooltip: classes.customWidth }}
-                          interactive
-                          disableFocusListener
-                          disableTouchListener
-                          title={
-                            canUseStaff
-                              ? 'Staff Timesheets Info' // need to update for staff
-                              : // eslint-disable-next-line no-multi-str
-                                'Student employees are not permitted to work more than 20 total hours\
+    if (networkStatus === 'online' && isUserStudent && props.Authentication) {
```

Comment:
I've changed the condition statement here. In the new condition statement, we check to make sure the user is logged in, a student, and online before displaying the Timesheets page. If the user is a staff, they will see a MUI card saying that Timesheets is unavailable. For both students and non-students, if they're offline, they will see a MUI card saying that they're offline.